src/DatePicker/CalendarActionButtons.js

```diff
@@ -32,20 +48,22 @@ class CalendarActionButton extends Component {
     };
 
     return (
-      <div style={styles.root} >
+      <div style={styles.root}>
         <FlatButton
+          className={cancelClassName}
           label={cancelLabel}
           onTouchTap={this.props.onTouchTapCancel}
           primary={true}
-          style={styles.flatButtons}
+          style={{...styles.flatButtons, ...cancelStyle}}
```

Comment:
sure thing!

Is there any particular reason you've picked `Object.assign` over spread though?